Tools/clinic/clinic.py

```diff
@@ -1790,7 +1795,7 @@ def print_block(self, block, *, core_includes=False):
                 #if defined(Py_BUILD_CORE) && !defined(Py_BUILD_CORE_MODULE)
                 #  include "pycore_gc.h"            // PyGC_Head
                 #  include "pycore_runtime.h"       // _Py_ID()
-                #endif
+                #endif  // Py_BUILD_CORE && !Py_BUILD_CORE_MODULE
```

Comment:
The block is small and clear, so I wasn't going to worry about that trailing comment.

Reply:
Thinking now I agree.